common/sys/platform.h

```diff
@@ -99,7 +99,7 @@
 #define dll_import 
 #endif
 
-#ifdef __WIN32__
+#if defined(__WIN32__) && !defined(__MINGW32__)
```

Comment:
This was needed to fix a crash issue when compiling Embree with MinGW-GCC, which we experienced upstream: https://github.com/godotengine/godot/issues/45097